src/presets/interfaces/account.cairo

```diff
@@ -0,0 +1,35 @@
+use starknet::account::Call;
+use starknet::{ContractAddress, ClassHash};
+
+#[starknet::interface]
+trait IAccountUpgradeable<TState> {
```

Comment:
```suggestion
trait AccountUpgradeableABI<TState> {
```

I think this is more accurate and consistent, wdyt? If we agree we need to update the rest as well.